tools/qualification-tool/pom.xml

```diff
@@ -0,0 +1,205 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.gluten</groupId>
+  <artifactId>qualification-tool</artifactId>
+  <version>1.3.0-SNAPSHOT</version>
+  <name>Qualification Tool</name>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <encoding>UTF-8</encoding>
+    <scala.version>2.12.15</scala.version>
```

Comment:
For all the configs which can take the value from top level pom, can we do that?

Reply:
Inheriting values from the parent POM resulted in some classes not being included in the tool. This issue occurred because the parent POM excludes certain classes from the Spark JARs that are required by this tool. Overriding the parent Spark dependencies to include those classes proved to be challenging. As a result, I decided to exclude the parent POM.